docs/src/componentDocs/Drawer/markdown/DrawerAPIDocs.mdx

```diff
@@ -24,10 +24,20 @@ The `<Drawer>` component is a wrapper around the [MUI Drawer](https://mui.com/ma
         src={Drawer}
     />
 </Box>
+
 ## Drawer
 
 The `<Drawer>` component is the parent container, which manages the overall state of the drawer and renders the child components.
 
+A `<Drawer />` can be broken down into the following subsections:
+
+-   [`<DrawerHeader />`](/components/drawer/drawer-header/examples)
+-   [`<DrawerSubheader />`](/components/drawer-sub-header/examples)
+-   [`<DrawerBody />`](/components/drawer-body/examples)
+-   [`<DrawerNavItem />`](/components/drawer/drawer-nav-item/examples)
+-   [`<DrawerNavGroup />`](/components/drawer/drawer-nav-group/examples)
+-   [`<DrawerFooter />`](/components/drawer-footer/examples)
```

Comment:
Some links have a `/drawer/` section in their URL, some do not.

Reply:
Added this to cleanup story [BLUI-3487](https://jira-prod.tcc.etn.com/browse/BLUI-3487)